test/test.js

```diff
@@ -97,8 +98,7 @@ describe("s3", function () {
       Prefix: remoteRoot,
       Bucket: s3Bucket,
     };
-    var deleter = client.deleteDir(s3Params);
-    deleter.on('end', function() {
+    client.createBucket(s3Bucket, function() {
```

Comment:
There is the S3_ENDPOINT env var that one can use to determine what ... endpoint to use.

I'm not sure of the spec for createBucket - does it fail catastrophically if one attempts to create a bucket that already exists?